File: PHP/MySQL/delete.php
<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<title>Test</title>

<?php

// DB接続
$db = new mysqli('localhost', 'root', 'password', 'sarvant');

if (!$db) {
    print("DB接続失敗<br>");
}

print("DB接続成功<br>");

// データ追加
$cmd = 'INSERT INTO artoria(name) VALUES("nero claudius")';
$result1 = mysqli_query($db, $cmd);
if (!$result1) {
    print("データ追加に失敗<br>");
}

// データ削除
$cmd = 'DELETE FROM artoria WHERE name = "nero claudius"';
$result2 = mysqli_query($db, $cmd);
if (!$result2) {
    print("データ取得に失敗<br>");
}

// データ取得
$cmd = 'SELECT name FROM artoria';
$result3 = mysqli_query($db, $cmd);
while ($row = mysqli_fetch_assoc($result3)) {
    print($row['name'].'<br>');
}

// DB切断
$closed_flag = mysqli_close($db);

if (!$closed_flag){
    print("DB切断失敗<br>");
}

print("DB切断成功<br>");

?>
</head>
  
<body>
</body>
</html>

File: Python/MySQL/mysql-connector/delete.py
# -*- coding: utf-8 -*-
import mysql.connector

def main():
    # MySQL接続
    cnt = mysql.connector.connect(
        host='localhost',
        port='3306',
        db='sarvant',
        user='root',
        password='password',
        charset='utf8'
    )

    # カーソル取得
    db = cnt.cursor(buffered=True)

    # SQLクエリ実行（データ取得）
    sql = 'DELETE FROM artoria WHERE name = "nero claudius";'
    db.execute(sql)

    # 表示
    rows = db.fetchall()
    print(rows)

    # カーソル終了
    db.close()
    # MySQL切断
    cnt.close()

if __name__ == "__main__":
    main()
